Formula/vespa-cli.rb

```diff
@@ -0,0 +1,32 @@
+class VespaCli < Formula
+  desc "Command-line tool for Vespa.ai"
+  homepage "https://vespa.ai"
+  url "https://github.com/vespa-engine/vespa/archive/refs/tags/vespa-7.462.20-1.tar.gz"
```

Comment:
same artifact, but shorter url
```suggestion
  url "https://github.com/vespa-engine/vespa/archive/vespa-7.462.20-1.tar.gz"
```